finalPrep/modZeroHero.js

```diff
@@ -56,16 +108,54 @@ var excuse = 'I think I forgot to lock up my 1992 Toyota Coralla. Be right back.
 //   - luckyNumbers (array)
 //   - address (object with following key/values: number , street , state, zip)
 
+var scaryMonster = {
+  name: "Mr. Gobblin",
+  smell: "musty",
+  weight: 300,
+  citiesDestroyed: ["Denver", "St. Louis", "Chicago"],
+  luckyNumbers: [666, 420, 69],
+  address: {
+    number: 12110,
+    street: "Devil Lane",
+    state: "Colorado",
+    zipCode: 80211
+  }
+};
 
 // Create a new class called SuperHero
 // - Your class should have the following DYNAMIC values
-//   - name 
+//   - name
 //   - superpower
-//   - age 
+//   - age
 // - Your class should have the following STATIC values
 //   - archNemesis, assigned to "The Syntax Error"
 //   - powerLevel = 100
-//   - energyLevel = 50 
+//   - energyLevel = 50
+
+class SuperHero {
+  constructor(name, superpower, age, archNemesis, powerLevel, energyLevel) {
+    this.name = name;
+    this.superpower = superpower;
+    this.age = age;
+    this.archNemesis = "The Syntax Error";
+    this.powerLevel = 100;
+    this.energyLevel = 50;
+  }
+
+  sayName() {
+    console.log(this.name);
+  }
+
+  maximizeEnergy() {
+    this.energyLevel = 1000;
+  }
+
+  gainPower(gainPower = 20) {
```

Comment:
here, you have set a default value to use when no argument is passed in your `gainPower` method. However, you can also pass a value as an argument to make a power gain of a different amount. One tip- I would name the argument something different than the method.